ravindu14/SliitSpmProject
# Feature/task id #1165 upload download buttons

how to test
1. go to 'http://localhost:4200/blank-page' url
2. fill all the feilds and click on search button (upload and download buttons should be visible)
3. when clicking on save button message should be appeared
